fortanix-sgx-tools/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fortanix-sgx-tools"
-version = "0.4.3"
+version = "0.5.0"
```

Comment:
This can be a patch bump

Reply:
The behavior is changed though. It now doesn't pass the filename as first arg